New package: AnyLogic.AnyLogic.Personal version 8.7.12

mdanish-kh opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----


###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/66121)

Commit:
New package: AnyLogic.AnyLogic.PLE version 8.7.12

Trenly commented on manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+- Architecture: x64
+  InstallerUrl: https://files.anylogic.com/anylogic-ple-8.7.12.x86_64.exe
+  InstallerSha256: E13F6AAB83A191B5A892325FEA112E2F4FD9618BA176ECF339F3E9D2C2DCF2CF
+- Architecture: x86
+  InstallerUrl: https://files.anylogic.com/anylogic-ple-8.7.12.x86_64.exe
+  InstallerSha256: E13F6AAB83A191B5A892325FEA112E2F4FD9618BA176ECF339F3E9D2C2DCF2CF
```
```suggestion
```
![image](https://user-images.githubusercontent.com/12611259/178646326-dbfe21f7-ab63-4248-bd06-dde2028b202b.png)

mdanish-kh replied:
Oh, I assumed the installer works for x86 as the name of the installer suggested so. Would check this next time in a 32-bit VM.

Trenly commented on manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.locale.en-US.yaml:
```diff
@@ -0,0 +1,34 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.PLE
```
```suggestion
PackageIdentifier: AnyLogic.AnyLogic.Personal
```

mdanish-kh replied:
Are there any good naming conventions for this? Or should I go by the package name in Control panel?

Trenly commented on manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.PLE
```
```suggestion
PackageIdentifier: AnyLogic.AnyLogic.Personal
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.yaml:
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.7-2-5
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: AnyLogic.AnyLogic.PLE
```
```suggestion
PackageIdentifier: AnyLogic.AnyLogic.Personal
```

Trenly commented on manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.locale.en-US.yaml:
```diff
@@ -0,0 +1,34 @@
+# PublisherSupportUrl: 
+PrivacyUrl: https://anylogic.com/upload/privacy_policy/anylogic-privacy-policy.pdf
+# Author: 
+PackageName: Anylogic Personal Learning Edition
```
Because they put the version in the middle of the display name, AppsAndFeaturesEntries are required

Trenly commented on manifests/a/AnyLogic/AnyLogic/PLE/8.7.12/AnyLogic.AnyLogic.PLE.installer.yaml:
```diff
@@ -0,0 +1,22 @@
+FileExtensions:
+- alp
+- java
+ReleaseDate: 2022-05-25
```
```suggestion
ReleaseDate: 2022-05-25
AppsAndFeaturesEntries: 
- DisplayName: AnyLogic 8.7.12 Personal Learning Edition
```

Commit:
Fix AnyLogic.Anylogic.Personal version 8.7.12

- Change package name 'PLE' to 'Personal'
- Remove x86 installer entry
- Add AppsAndFeaturesEntries
- Rename directory and manifest files
